price-estimator/docker/deploy.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
I like this in its own sub-directory! Maybe (probably in a separate PR) we should do this for the driver as well?

Reply:
I agree that we should do this.